lib/workers/repository/update/pr/body/config-description.ts

```diff
@@ -51,7 +54,8 @@ function scheduleToString(
 ): string {
   let scheduleString = '';
   if (schedule && schedule[0] !== 'at any time') {
-    scheduleString += `"${String(schedule)}"`;
+    scheduleString =
+      getReadableCronSchedule(schedule) ?? `"${String(schedule)}"`;
```

Comment:
```suggestion
      getReadableCronSchedule(schedule) ?? `"${schedule}"`;
```
do we really need this string cast?

Reply:
Yes, it converts the later schedule in array form to string separated by commas.